### DISCUSSION TITLE
Skywalking compile error, a protocol properties cannot find.AccessLogServiceHandler hasNode() getNode()

### DISCUSSION THREAD

**Discussion by Nonlinearequation** [apache/skywalking#12052] [Q&A]
Description
When I'm building skywalking source code,there is a compiling error.Idea prompt me that 
org.apache.skywalking.oap.server.receiver.ebpf.provider.handler.AccessLogServiceHandler can not find method of hasNode and getNode which belong to EBPFAccessLogMessage.

skywalking version 9.7.0
protocol buffers idea plugin version bundled 232.8660.185
maven 3.8.1
jdk 17
idea 2023.2

follow the step of  How-to-build.md

![企业微信截图_17115047877433](https://github.com/apache/skywalking/assets/26458273/b634b493-5ac5-400c-8947-c5f4a6593f3f)

**Comment by wu-sheng** [comment]
How do you download the source codes? Are you using `git clone`?

**Reply by Nonlinearequation** [reply]
yes，clone from skywalking apache https://github.com/apache/skywalking/tree/master .
And the code for the relevant module is downloaded using git submodule.

The EBPFAccessLogMessage class has no node field. Why AccessLogServiceHandler using this field?

![image](https://github.com/apache/skywalking/assets/26458273/1b14dd26-0e5b-44bb-b3f1-53cb17da00f4)

**Reply by Nonlinearequation** [reply]
Sorry, I just looked at the source code for this class, and the source code has the node field.
I'll try to compiling later.

https://github.com/apache/skywalking-data-collect-protocol/blob/f2f3005155339b09714b963e19fea5151d0d6d0b/ebpf/accesslog.proto
